Lib/fontTools/varLib/instancer/__init__.py

```diff
@@ -1413,16 +1421,41 @@ def parseArgs(args):
     if len(axisLimits) != len(options.locargs):
         parser.error("Specified multiple limits for the same axis")
 
+    if axisLimits and options.named_instances:
+        parser.error("Specified limits and --fvar")
+
     return (infile, axisLimits, options)
 
 
 def main(args=None):
     """Partially instantiate a variable font."""
     infile, axisLimits, options = parseArgs(args)
-    log.info("Restricting axes: %s", axisLimits)
 
     log.info("Loading variable font")
     varfont = TTFont(infile)
+    if options.named_instances:
+        for instance in varfont["fvar"].instances:
+            name = varfont["name"].getDebugName(instance.subfamilyNameID)
+            if not re.search(options.named_instances, name):
+                continue
+            axisLimits = instance.coordinates
+            log.info("Restricting axes: %s (%s)", axisLimits, name)
+            name = name.replace(" ", "")
+            outfile = os.path.splitext(infile)[0] + "-" + name + ".ttf"
```

Comment:
the -o/--output option is also incompatible with this new --named-instances, because the latter usually outputs multiple files, not a single one

Reply:
If `--output` is used this code should error, or, if it wants to be smart, use it for the first matching instance and error for the rest, so one can use it to output a single named instance (which happens to be my use case, since I use Makefile and rules that output multiple files is PITA).